KsApi/fragments/ProjectFragment.graphql

```diff
@@ -94,4 +94,5 @@ fragment ProjectFragment on Project {
     }
   }
   watchesCount
+  isPledgeOverTimeAllowed
```

Comment:
This fragment currently has fields in (approximately) alphabetical order - would you mind moving this one up to all the other `is_`s?

Reply:
Done